aws-lightsail-loadbalancertlscertificate/src/main/java/software/amazon/lightsail/loadbalancertlscertificate/helpers/GetModifiedLbTlsCertResponse.java

```diff
@@ -0,0 +1,38 @@
+package software.amazon.lightsail.loadbalancertlscertificate.helpers;
+
+import software.amazon.awssdk.services.lightsail.model.GetLoadBalancerTlsCertificatesResponse;
+
+/**
+ * We are using the LoadBalancerTlsCertificate resource type to modify the HttpsRedirectionEnabled parameter of the LoadBalancer.
```

Comment:
Could you provide more detail as to why we're doing this? It seems weird to use another resource to modify a property that's defined on another resource.